src/main/java/io/reactivex/Scheduler.java

```diff
@@ -440,4 +431,23 @@ public boolean isDisposed() {
             return disposed;
         }
     }
+
+    private static class DelegationTask implements Runnable {
```

Comment:
Please avoid the `private` modifier as it usually generates extra accessor methods.